misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/BTestRunner.java

```diff
@@ -202,20 +205,22 @@ private void compileAndBuildSuites(String sourceRoot, Path[] sourceFilePaths, bo
      */
     private void execute(boolean buildWithTests) {
         Map<String, TestSuite> testSuites = registry.getTestSuites();
+        outStream.println();
+        outStream.println("Running tests");
         if (testSuites.isEmpty()) {
+            outStream.println("    No tests found");
```

Comment:
Should we use \t instead of the spaces ? Please check against other println calls

Reply:
This space is not equal to the tab space.

Reply:
👍